server/services/__testutils/mocks.ts

```diff
@@ -7,12 +7,13 @@ import {
   TemporaryAbsencesService,
   UserService,
   NotificationService,
+  MatchTypeDecorator,
 } from '..'
 
 jest.mock('..')
 
 export const createMockExpectedArrivalsService = () =>
-  new ExpectedArrivalsService(null, null, null, null) as jest.Mocked<ExpectedArrivalsService>
+  new ExpectedArrivalsService(null, null, null, null, null) as jest.Mocked<ExpectedArrivalsService>
```

Comment:
This centralising of mock servers has really paid off - had to change 30 odd test files last time we add a dependency to this service!